### ISSUE TITLE
python3 iohub.launchHubServer() error

### ISSUE THREAD

**Issue by tjwilli58** [psychopy/psychopy#1760]
When I try to do iohub.launchHubServer(), I get an error

```
io=iohub.launchHubServer()

Traceback (most recent call last):

  File "<ipython-input-3-8e0d9059692b>", line 1, in <module>
    io=iohub.launchHubServer()

  File "C:\Users\Tim\Anaconda3\lib\site-packages\psychopy\iohub\client\connect.py", line 301, in launchHubServer
    return ioHubConnection(iohub_config)

  File "C:\Users\Tim\Anaconda3\lib\site-packages\psychopy\iohub\client\__init__.py", line 286, in __init__
    self.iohub_status = self._startServer(ioHubConfig, ioHubConfigAbsPath)

  File "C:\Users\Tim\Anaconda3\lib\site-packages\psychopy\iohub\client\__init__.py", line 853, in _startServer
    with file(cfpath, 'r') as config_file:

NameError: name 'file' is not defined

```
Need to change 'file' to 'open' in iohub/client/__init__.py
```
        # >>>>> Load / Create / Update iohub config file.....

        cfpath = os.path.join(IOHUB_DIRECTORY, 'default_config.yaml')
        with file(cfpath, 'r') as config_file:
            hub_defaults_config = yload(config_file, Loader=yLoader)

```
to
```
        # >>>>> Load / Create / Update iohub config file.....

        cfpath = os.path.join(IOHUB_DIRECTORY, 'default_config.yaml')
        with open(cfpath, 'r') as config_file:
            hub_defaults_config = yload(config_file, Loader=yLoader)

```
likewise

```
        elif ioHubConfigAbsPath is not None and ioHubConfig is None:
            with open(ioHubConfigAbsPath, 'r') as config_file:
                ioHubConfig = yload(config_file, Loader=yLoader)

```

**Comment by tjwilli58** [comment]
I still get errors, but I get a little further:

```
io=iohub.launchHubServer()
Traceback (most recent call last):

  File "<ipython-input-2-57616307f219>", line 1, in <module>
    io=iohub.launchHubServer()

  File "C:\Users\Tim\Anaconda3\lib\site-packages\psychopy\contrib\lazy_import.py", line 120, in __call__
    return obj(*args, **kwargs)

  File "C:\Users\Tim\Anaconda3\lib\site-packages\psychopy\iohub\client\connect.py", line 301, in launchHubServer
    return ioHubConnection(iohub_config)

  File "C:\Users\Tim\Anaconda3\lib\site-packages\psychopy\iohub\client\__init__.py", line 288, in __init__
    raise RuntimeError('Error starting ioHub server')

RuntimeError: Error starting ioHub server


```
Not sure how to track this one down. Sorry.
Thanks for any help.

**Comment by peircej** [comment]
Thanks for the fixes you did find. Yes, switching from `file` to `open` is what we need.
For the other issue I don't know either but will try and test it out tomorrow

**Comment by peircej** [comment]
I think all these were fixed in commits 513a6daf3a07c62b3cc689430f0a393881b3af7c and c4fb92b3cd6a419e214f261680ad1e949fd93f60
